en/rules.md

```diff
@@ -3,7 +3,7 @@ title: "Rules"
 old_id: 9
 ---
 <h4 class="centered">Rules may be changed at any time as we see fit.<br>New rules may or may not be retroactive at our discretion.</h4>
-<h4 class="centered">Rules on this page may not always be updated. [Join our Discord](https://discord.gg/4CzsqkK) To guarantee you're reading the right rules. These rules were updated on October 13th, 2022.</h4>
+<h4 class="centered">Rules on this page may not always be updated. Join our Discord to guarantee you're reading the right rules. These rules were updated on December 31st, 2022.</h4>
```

Comment:
This may sound better > The rules on this page may not always be updated. Join our Discord to guarantee you're reading the right rules. This rule's last update was on December 31st, 2022.

Also, let's try to not put hard words in the rules section so everyone can understand them easily.

Reply:
> This may sound better > The rules on this page may not always be updated. Join our Discord to guarantee you're reading the right rules. This rule's last update was on December 31st, 2022.
> 
> Also, let's try to not put hard words in the rules section so everyone can understand them easily.

`This rule's last update was on December 31st, 2022.` sounds like you're referring to only one rule tbh

Reply:
I am referring to website rules only because sometimes the discord and website rules ain't updated at the same time. Or maybe you can mean referring to just one rule? if so then I guess you can say "this rules" instead

Reply:
> 

you mean `these rules` 🗿🗿

Reply:
yeah you are right